amgut/lib/data_access/env_management.py

```diff
@@ -132,8 +133,12 @@ def make_settings_table():
 def populate_test_db():
     conn = SQLConnectionHandler()
 
-    with open(POPULATE_FP) as f:
-        conn.execute(f.read())
+    with gzip.open(POPULATE_FP, 'rb') as f:
+        test_db = f.read()
+
+    command = ['psql', '-d', AMGUT_CONFIG.database]
+    proc = Popen(command, stdin=PIPE, stdout=PIPE)
+    proc.communicate(test_db)
```

Comment:
Yeah, tried just psycopg2 but the way pd_dump gives files needs to be loaded with psql.